VTEX - Audience API.json

```diff
@@ -0,0 +1,320 @@
+                                "example": {
+                                    "error": "Invalid input or account not detected."
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/api/price-table-mapper/pvt/mapping/{audienceId}": {
+            "get": {
+                "tags": [
+                    "Price Table Mapper"
+                ],
+                "summary": "Get price table mapping",
+                "description": "Fetches the price table mapping for a specific audience.\r\n\r\n## Permissions\r\nThis endpoint requires any one of the [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy).",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/AudienceParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string",
+                                        "description": "Price table."
+                                    }
+                                },
+                                "example": [
+                                    "priceTable01"
+                                ]
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/responseError"
+                                },
+                                "example": {
+                                    "error": "Mapping not found for the provided audienceId."
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Price Table Mapper"
+                ],
+                "summary": "Set price table mapping",
+                "description": "Sets the price table mapping for the provided audience.\r\n\r\n## Permissions\r\nThis endpoint requires any one of the [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy).",
```

Comment:
- What does this mean aside from repeating the summary? I'm guessing it could be more like "Defines the price table that apply to the provided audience."
- Can it be more than one price table? (given that it's an array) If yes, how do they relate to each other?